WordPress/Classes/ViewRelated/Reader/WPImageViewController.m

```diff
@@ -5,9 +5,9 @@
 
 @interface WPImageViewController ()<UIScrollViewDelegate>
 
-- (void)handleImageTapped:(UITapGestureRecognizer *)tgr;
-- (void)handleImageDoubleTapped:(UITapGestureRecognizer *)tgr;
-
+@property (nonatomic) BOOL isLoadingImage;
```

Comment:
I agree with the consistency, but I think we should not be using `assign` or `readwrite` anywhere in the code, since they are the default option. Just my 2 cents.